test-e2e/utils/steps_file.ts

```diff
@@ -90,7 +177,7 @@ module.exports = function() {
       // Since this check executes a script in the browser, it won't use the codecept retries,
       // so we have to manually retry (this is because the video can take time to load and the state will be buffering)
       for (let i = 0; i < tries; i++) {
-        const state = await this.executeScript(() => typeof jwplayer === 'undefined' ? '' : jwplayer().getState());
+        const state = await this.executeScript(() => typeof jwplayer === 'undefined' || typeof jwplayer().getState === 'undefined' ? '' : jwplayer().getState());
```

Comment:
Can we maybe have directly `!jwplayer || !jwplayer().getState` instead of `typeof jwplayer === 'undefined'`? 
If `jwplayer` variable will be `undefined`, it will be a `falsy` value
Maybe you can try to see if it still works with this adjustment 🙂

Reply:
That should work, but something about the way that codeceptjs processes the page.evaluate causes the error below unless I write it exactly like this.  I think it's worth putting a comment for the next person though. Thanks for pointing it out.
```
 page.evaluate: ReferenceError: jwplayer is not defined
    at eval (eval at evaluate (:3:2389), <anonymous>:1:7)
    at t.default.evaluate (<anonymous>:3:2412)
    at t.default.<anonymous> (<anonymous>:1:44)
```